src/Compilers/CSharp/Test/Emit2/Semantics/ExtensionTypeTests.cs

```diff
@@ -22943,32 +23087,21 @@ public class C
 """;
         var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
         // PROTOTYPE should warn about hiding
-        // PROTOTYPE update overload resolution so that extension members can hide members of the underlying type
-        comp.VerifyDiagnostics(
-            // (12,31): error CS0121: The call is ambiguous between the following methods or properties: 'E.M2()' and 'C.M2()'
-            //     public void M(E e) { this.M2(); e.M2(); M2(); }
-            Diagnostic(ErrorCode.ERR_AmbigCall, "M2").WithArguments("E.M2()", "C.M2()").WithLocation(12, 31),
-            // (12,39): error CS0121: The call is ambiguous between the following methods or properties: 'E.M2()' and 'C.M2()'
-            //     public void M(E e) { this.M2(); e.M2(); M2(); }
-            Diagnostic(ErrorCode.ERR_AmbigCall, "M2").WithArguments("E.M2()", "C.M2()").WithLocation(12, 39),
-            // (12,45): error CS0121: The call is ambiguous between the following methods or properties: 'E.M2()' and 'C.M2()'
-            //     public void M(E e) { this.M2(); e.M2(); M2(); }
-            Diagnostic(ErrorCode.ERR_AmbigCall, "M2").WithArguments("E.M2()", "C.M2()").WithLocation(12, 45)
-            );
+        comp.VerifyDiagnostics();
```

Comment:
>comp.VerifyDiagnostics(); [](http://example.com/codeflow?start=8&length=25)

Run the code?

Reply:
PROTOTYPE comment was already meant to indicate that, but updated it to be unambiguous